test/e2e/network/dns_common.go

```diff
@@ -169,6 +180,10 @@ func (t *dnsTestCommon) deleteConfigMap() {
 }
 
 func (t *dnsTestCommon) createUtilPod() {
```

Comment:
Can't you just get rid of this function or replace it's uses with `createUtilPodLabel`?